drivers/power/supply/qcom_fg.c

```diff
@@ -1253,6 +1257,59 @@ static void qcom_fg_remove(struct platform_device *pdev)
 		destroy_workqueue(chip->sram_wq);
 }
 
+static int qcom_fg_suspend(struct device *dev)
+{
+	bool sram_accessible;
+	unsigned long flags;
+	int sram_requests;
+	int ret;
+	struct qcom_fg_chip *chip = dev_get_drvdata(dev);
+
+	spin_lock_irqsave(&chip->sram_request_lock, flags);
+	sram_accessible = qcom_fg_sram_check_access(chip);
+	sram_requests = chip->sram_requests;
+	spin_unlock_irqrestore(&chip->sram_request_lock, flags);
+
+	if(sram_requests > 0 && !sram_accessible)
+	{
+		/* Wait to get access to SRAM, and try again if interrupted */
+		do {
+			ret = wait_for_completion_interruptible_timeout(
+				&chip->sram_access_granted,
+				msecs_to_jiffies(MEM_IF_TIMEOUT_MS));
+		} while(ret == -ERESTARTSYS);
+
+		if (ret <= 0)
+		{
+			dev_err(chip->dev,
+						"Failed to sync SRAM state: %d\n", ret);
+			return -ETIMEDOUT;
+		}
+	}
+
+	cancel_delayed_work_sync(&chip->sram_release_access_work);
+
+	return 0;
+}
+
+static int qcom_fg_resume(struct device *dev)
+{
+	struct qcom_fg_chip *chip = dev_get_drvdata(dev);
+	spin_lock(&chip->sram_request_lock);
+
+	if(chip->sram_requests == 0)
+		/* Schedule access release */
+		queue_delayed_work(chip->sram_wq, &chip->sram_release_access_work,
+			msecs_to_jiffies(SRAM_ACCESS_RELEASE_DELAY_MS));
```

Comment:
Please align the parameters

Reply:
it will be more than 80 chars https://www.kernel.org/doc/html/v5.6/process/coding-style.html

Reply:
If you put `chip->sram_wq` in a new line then you can align them